src/widgets/Menu.tsx

```diff
@@ -0,0 +1,28 @@
+// SPDX-FileCopyrightText: © 2017 EteSync Authors
+// SPDX-License-Identifier: AGPL-3.0-only
+
+import * as React from 'react';
+
+import MuiMenu, { MenuProps } from '@material-ui/core/Menu';
+import { PopoverOrigin } from '@material-ui/core/Popover';
+
+const anchorOrigin: PopoverOrigin = {
+  vertical: 'bottom',
+  horizontal: 'right',
+};
+
+const transferOrigin: PopoverOrigin = {
+  vertical: 'top',
+  horizontal: 'right',
+};
+
+export default function Menu(props: MenuProps) {
+  return (
+    <MuiMenu
+      {...props}
+      getContentAnchorEl={null}
```

Comment:
What's this, and why is it needed? I'd expect it to do something fancy like checking the menu fits the screen in this specific orientation. Make sure that's not broken.

Reply:
It's to surpress this error:
```
you can not change the default `anchorOrigin.vertical` value 
when also providing the `getContentAnchorEl` prop to the popover component.
Only use one of the two props.
Set `getContentAnchorEl` to `null | undefined` or leave `anchorOrigin.vertical` unchanged.
```

Honestly not sure what is going on here because I am not manually providing the `getContentAnchorEl` props. Must have something to do with how `Menu` uses the `Popover` component internally. I just followed what it said and nothing breaks.

Reply:
See https://github.com/mui-org/material-ui/blob/3a64fe16ff1bd3a72bf0d49d74eba280a839c2eb/packages/material-ui/src/Menu/Menu.js#L62

Reply:
Cool, I believe you. All good. :)